pom.xml

```diff
@@ -78,10 +78,10 @@
   <modules>
     <module>appengine-java-logging</module>
     <module>appengine-managed-runtime</module>
-    <module>gke-debian-openjdk</module>
-    <module>gke-jetty</module>
+    <module>openjdk8</module>
+    <module>jetty9</module>
```

Comment:
I think the gke- suffix is meaningful in this context because it is a docker image customise for running in GKE (debuggers, loggers etc.) with jetty9.  Just calling the artefact jetty9 is a little over brief to me